modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -31,6 +31,24 @@ module LocaleModel {
   use LocaleModelHelpFlat;
   use LocaleModelHelpMem;
 
+  //
+  // The task layer calls these to convert between full sublocales and
+  // execution sublocales.  Full sublocales may contain more information
+  // in some locale models, but not in this one.
+  //
+  export
+  proc chpl_localeModel_sublocToExecutionSubloc(full_subloc:chpl_sublocID_t)
```

Comment:
Given that we've resolved that there is currently a need for calling into the locale model from the tasking layer, the remaining question about the chpl_localeModel_* functions is their long names.  I could make it _lm_ instead of _localeModel_.  I just wanted to be sure that someone reading the tasking code would know exactly where to look when they came across those calls.  I'm not sure _lm_ is enough to do that, but I'm open to suggestions.